src/common/utility.js

```diff
@@ -179,4 +179,23 @@ export const getLocalizedUrl = (path, is_index, to) => `/${path}${is_index ? ``
 export const nonENLangUrlReplace = (current_path) => {
     const path_with_or_without_slash = /\/.+?(\/)|(\/[a-zA-Z'-]+)/u
     return current_path.replace(path_with_or_without_slash, '')
-}
\ No newline at end of file
+}
+
+export const getDateFromToday = (num_of_days) => {
+    const today = new Date()
+    const end_date = new Date(today.getFullYear(), today.getMonth(), today.getDate() + num_of_days)
+
+    return end_date
+}
+
+export const isNullUndefined = (value) => value === null || value === undefined
+
+export const isObject = (value) => typeof value === 'object'
+
+export const isJSONString = (value) => {
```

Comment:
Check comment below~

Reply:
i think its better to check isJSONString and parseJSON and make it as isolated as possible, so it can be used in any type of situation